docs/en/stores/basic-usage.md

```diff
@@ -0,0 +1,454 @@
+}
+
+export interface State {
+	auth: {
+		token: string;
+	};
+	users: {
+		current: User;
+		list: User[];
+	};
+}
+```
+
+The above is a simple example for illustrative purposes. Examples in this reference guide will refer back to this interface to reduce repetition.
+
+## Updating stores
+
+To work with Dojo Stores there are three core concepts.
+
+-   **Operations** are instructions to manipulate the state held by the store
+-   **Commands** are simple functions that perform business logic and return operations
+-   **Processes** execute a group of commands and represent application behavior
+
+### Commands & Operations
+
+Stores cannot be modified directly. Instead, operations that modify the store are returned from commands, which are simply functions that are called while executing a process. Each command is passed a `CommandRequest` which provides `path` and `at` functions to generate `Path`s in a type-safe way, a `get` function for access to the store's state, a `payload` object for the argument that the process executor was called with.
+
+#### The command factory
+
+Stores have a simple wrapper function that acts as a type-safe factory for creating new commands.
+
+> Creating a store factory
+
+```ts
+import { createCommandFactory } from '@dojo/framework/stores/process';
+import { State } from './interfaces';
+
+const createCommand = createCommandFactory<State>();
+
+const myCommand = createCommand(({ at, get, path, payload, state }) => {
+	return [];
+});
+```
+
+`createCommand` will ensure that the wrapped command has the correct typing and the passed `CommandRequest` functions will be typed to the `State` interface provided to `createCommandFactory`. While it is possible to manually type commands this guide will use `createCommand` in its examples.
+
+#### `path`
+
+The path is a `string` that describes the location where an operation is applied. The `path` function is part of the `CommandRequest` and is accessible inside of a `Command`.
+
+Here is an example to show how `path` describes a location in the store. The `State` is the same as defined above in `interface.d.ts`. It is used by the `Store` to understand the shape of the state data.
+
+> `path` of the current user name
+
+```ts
+const store = new Store<State>();
+const { path } = store;
+
+path('users', 'current', 'name');
+```
+
+This path refers to the `string` value located at `/users/current/name`. `path` is used to transverse the hierarchy in a type-safe way. It ensures only the property names defined in the `State` interface are used.
+
+#### `at`
+
+The `at` function is used in conjunction with `path` to identify a location in an array. Here is an example of using the `at` function.
+
+```ts
+const store = new Store<State>();
+const { at, path } = store;
+
+at(path('users', 'list'), 1);
+```
+
+This path refers to the `User` located at `/users/list` at offset `1`.
+
+#### `add` operation
+
+Adds a value to an object or inserts it into an array.
+
+> `add` example
```

Comment:
I think this formatting is used to indicate the filename not a title, perhaps @sbinge can confirm?